# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/Document.h
# Commit: 231b0b4efd16
# Full Hash: 231b0b4efd16880f3273859e7dc97b4cfe789c21
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2023-12-06 05:18:37
# Regressor Bug: 1589554
# File Overlap Count: 1
# Description:
#   Bug 1589554 - Part 2: Implement Screen Wake Lock API. r=dom-core,edgar
#   
#   Depends on D189508
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D189509
# ==============================================================================

diff -r f50ff29a1a64 -r 231b0b4efd16 dom/base/Document.h
--- a/dom/base/Document.h	Tue Dec 05 23:58:06 2023 +0000
+++ b/dom/base/Document.h	Tue Dec 05 23:58:06 2023 +0000
@@ -3540,6 +3540,8 @@
 
   nsTHashSet<RefPtr<WakeLockSentinel>>& ActiveWakeLocks(WakeLockType aType);
 
+  void UnlockAllWakeLocks(WakeLockType aType);
+
   // ParentNode
   nsIHTMLCollection* Children();
   uint32_t ChildElementCount();